deployment/env/base.yaml

```diff
@@ -0,0 +1,25 @@
+---
+image:
+  pullPolicy: Always
+
+serviceAccount:
+  create: true
+
+service:
+  type: NodePort
+  port: 3000
+
+ingress:
```

Comment:
Does nginx have access to the file system of the Rails container? If so it should be serving static assets from `/rails/public`

Reply:
tl;dr that is not possible

ingress controllers do not share user space with pods so don't have access to their files. This block is defining the Ingress k8s object that configures routing on controllers.

Reply:
So the solution would be start nginx in the same container as the rails web server